pages/linux/bmon.md

```diff
@@ -0,0 +1,19 @@
+# bmon
+
+> Monitor bandwidth and capture network related statistics.
+
+- Display the list of all the interfaces:
+
+`bmon -a`
+
+- Display rates in bits per second:
```

Comment:
rates of what ? May be good to clarify that.

Reply:
Updated the description